actix/src/SUMMARY.md

```diff
@@ -1,14 +1,16 @@
 # Summary
 
 [Actix Quickstart](./sec-0-quick-start.md)
+
 - [Getting Started](./sec-1-getting-started.md)
 - [Actor](./sec-2-actor.md)
 - [Address](./sec-3-address.md)
 - [Context](./sec-4-context.md)
 - [Arbiter](./sec-5-arbiter.md)
 - [SyncArbiter](./sec-6-sync-arbiter.md)
-- [Stream](./sec-7-stream.md)
-- [IO helpers](./sec-8-io-helpers.md)
-- [Supervisor](./sec-9-supervisor.md)
-- [Registry](./sec-10-registry.md)
-- [Helper actors](./sec-11-helper-actors.md)
+- [ActorFuture](./sec-7-actor-future.md)
```

Comment:
It would be great if we could take the numbers out of these filenames. I kind of get why they're there (because now the files are ordered), but it makes it a pain to insert sections. It would make more sense if there weren't 5 entirely WIP pages that will probably need to be reordered anyways.